package.json

```diff
@@ -82,7 +82,7 @@
   },
   "packageManager": "yarn@3.2.1",
   "engines": {
-    "node": ">=16.0.0"
+    "node": "^16.20 || ^18.16 || >=20"
```

Comment:
Small thing, but is it necessary to be so strict about the version of 16.x and 18.x? Does it make sense to use `^16 || ^18 || >=20` now, and then carry that trend forward, so when 22 comes out we shift it to `^18 || ^20 || >=22`? I suppose we could simplify this later, but if we can do it now then it seems better to me.

Reply:
We had a discussion about this over a call and also on Slack and have resolved this. This strategy makes sense provided that we now use this range as a template for other repos going forward (except where necessary to diverge).